### ISSUE TITLE
Breaks when used in HMR environment

### ISSUE THREAD

**Issue by pribilinskiy** [apexcharts/vue-apexcharts#10]
When HMR (Hot Module Replacement, Webpack 4) is enabled and an update is received the chart is not updated anymore and there are errors whenever a change in the layout happens, e.g. changing browser size.

```
Uncaught TypeError: Cannot read property 'querySelectorAll' of undefined
```

This can be easily reproduced just by using [the topmost basic component example in README](https://github.com/apexcharts/vue-apexcharts/blob/master/README.md)

![image](https://user-images.githubusercontent.com/3078595/45688671-b3099300-bb5a-11e8-900e-d4386c2c5b39.png)

**Comment by junedchhipa** [comment]
Cannot reproduce in Vue CLI 3.
Didn't tested in custom configuration environment. Can you provide more info or create a codesandbox demo, please?

**Comment by hdriqi** [comment]
I have the same problem. When first load, chart works find but when I navigate to other page and back to the chart page to generate new charts, the charts successfully generated but when hovered that error occurs. I think its failed to find the tooltip element.

I think the error occurs because of using the cached tooltip element which already destroyed when the page updated.

**Comment by hdriqi** [comment]
Turns out the error only happens when using the `group` in `chartOptions`. Any ideas?

**Comment by junedchhipa** [comment]
If this happens only when `group` is used, then the issue https://github.com/apexcharts/apexcharts.js/issues/230 is similar to this one. I have fixed that bug, you may test it after the next release

**Comment by hdriqi** [comment]
@junedchhipa thanks! just tell us when the next release is up

**Comment by junedchhipa** [comment]
Released apexcharts@2.4.2. Can you please download and test it?

**Comment by hdriqi** [comment]
@junedchhipa its working perfectly! thanks dude
